<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSectionDialog</class>
 <widget class="QDialog" name="AddSectionDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>596</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>10</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>GIAP-PolaMap(lite) - Add/Edit section</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/plugins/GIAP-giap_layout/icons/giap_logo.png</normaloff>:/plugins/GIAP-giap_layout/icons/giap_logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="zakoncz">
      <layout class="QGridLayout" name="gridLayout_9">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QFrame" name="frame_18">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: #4f5a63;
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <widget class="QLabel" name="title_label">
          <property name="geometry">
           <rect>
            <x>8</x>
            <y>0</y>
            <width>231</width>
            <height>30</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color : #EDF6FC;	font-weight: bold;</string>
          </property>
          <property name="text">
           <string>Add/Edit custom tool section</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="frame_main">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_17">
          <item row="1" column="1">
           <widget class="QFrame" name="frame">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border-bottom: 1px solid white;
border-left: none;
border-right: none;
border-top: none;</string>
               </property>
               <property name="text">
                <string>Available tools</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="availableToolTable">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="lineWidth">
                <number>0</number>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::MultiSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="showGrid">
                <bool>false</bool>
               </property>
               <property name="cornerButtonEnabled">
                <bool>false</bool>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>250</number>
               </attribute>
               <attribute name="horizontalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderMinimumSectionSize">
                <number>200</number>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="2">
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QToolButton" name="toolButton_add_tool">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>100</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>10</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton {
padding: 5px;
border: none;
border-width: 1px;
border-radius: 3px;
background-color: #5589B0;
}
QToolButton:checked {
background-color: #5589B0;
border: solid;
border-width: 1px;
border-color: #5589B0;
}

QToolButton:disabled  {
background-color: #465A68;
color: #7E7F81;
}

QToolButton:hover {
background-color: #5589B0;
border-radius: 3px;
border: solid;
border-width: 1px;
border-color: #E0DECF;
}</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="arrowType">
               <enum>Qt::RightArrow</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_remove_tool">
              <property name="minimumSize">
               <size>
                <width>30</width>
                <height>100</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton {
padding: 5px;
border: none;
border-width: 1px;
border-radius: 3px;
background-color: #5589B0;
}
QToolButton:checked {
background-color: #5589B0;
border: solid;
border-width: 1px;
border-color: #5589B0;
}

QToolButton:disabled  {
background-color: #465A68;
color: #7E7F81;
}

QToolButton:hover {
background-color: #5589B0;
border-radius: 3px;
border: solid;
border-width: 1px;
border-color: #E0DECF;
}</string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
              <property name="arrowType">
               <enum>Qt::LeftArrow</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="3">
           <widget class="QFrame" name="frame_6">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">border-bottom: 1px solid white;
border-left: none;
border-right: none;
border-top: none;</string>
               </property>
               <property name="text">
                <string>Selected tools</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTableView" name="selectedToolTable">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <property name="lineWidth">
                <number>0</number>
               </property>
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="dragDropMode">
                <enum>QAbstractItemView::NoDragDrop</enum>
               </property>
               <property name="defaultDropAction">
                <enum>Qt::IgnoreAction</enum>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
               <property name="showGrid">
                <bool>false</bool>
               </property>
               <property name="cornerButtonEnabled">
                <bool>false</bool>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>250</number>
               </attribute>
               <attribute name="horizontalHeaderHighlightSections">
                <bool>false</bool>
               </attribute>
               <attribute name="horizontalHeaderMinimumSectionSize">
                <number>200</number>
               </attribute>
               <attribute name="verticalHeaderVisible">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QToolButton" name="wdol_narzdzie">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QToolButton {
padding: 5px;
border: none;
border-width: 1px;
border-radius: 3px;
background-color: #5589B0;
}
QToolButton:checked {
background-color: #5589B0;
border: solid;
border-width: 1px;
border-color: #5589B0;
}

QToolButton:disabled  {
background-color: #465A68;
color: #7E7F81;
}

QToolButton:hover {
background-color: #5589B0;
border-radius: 3px;
border: solid;
border-width: 1px;
border-color: #E0DECF;
}</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="arrowType">
                  <enum>Qt::DownArrow</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="wgore_narzdzie">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>30</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QToolButton {
padding: 5px;
border: none;
border-width: 1px;
border-radius: 3px;
background-color: #5589B0;
}
QToolButton:checked {
background-color: #5589B0;
border: solid;
border-width: 1px;
border-color: #5589B0;
}

QToolButton:disabled  {
background-color: #465A68;
color: #7E7F81;
}

QToolButton:hover {
background-color: #5589B0;
border-radius: 3px;
border: solid;
border-width: 1px;
border-color: #E0DECF;
}</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="arrowType">
                  <enum>Qt::UpArrow</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1" colspan="3">
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item row="1" column="0">
              <widget class="QLabel" name="label_2">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Find tool</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Section name</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="section_name_lineedit"/>
             </item>
             <item row="1" column="1">
              <widget class="QgsFilterLineEdit" name="find_tool_searchbox">
               <property name="toolTip">
                <string>Enter algorithm name to filter list</string>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">color: #fff</string>
               </property>
               <property name="placeholderText">
                <string>Search...</string>
               </property>
               <property name="showSearchIcon">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <property name="spacing">
      <number>6</number>
     </property>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_save">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFilterLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgsfilterlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>AddSectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
